pkg/cmd/roachtest/tests/admission_control_elastic_backup.go

```diff
@@ -92,9 +93,13 @@ func registerElasticControlForBackups(r registry.Registry) {
 					m := c.NewMonitor(ctx, c.Range(1, crdbNodes))
 					m.Go(func(ctx context.Context) error {
 						t.Status(fmt.Sprintf("during: creating full backup schedule to run every 20m (<%s)", time.Minute))
-						_, err := db.ExecContext(ctx,
+                                                gcsBackupTestingBucket := os.Getenv("BACKUP_TESTING_BUCKET")
+                                                if gcsBackupTestingBucket == "" {
+                                                        gcsBackupTestingBucket="cockroach-bucket-testing-private"
```

Comment:
The defaults should continue to be the `cockroachdb-backup-testing` bucket, not the new one. The new one should be what is configured in the environment variable.